src/EnergyPlus/AirTerminalUnit.hh

```diff
@@ -94,7 +94,8 @@ protected: // Creation
 		airInNodeNum( 0 ),
 		airOutNodeNum( 0 ),
 		zoneIndex( 0 ),
-		zoneNodeIndex( 0 )
+		zoneNodeIndex( 0 ),
+		airLoopNum( 0 )
```

Comment:
I see I forgot this in #6248.  Also forgot to intialize ctrlZoneInNodeIndex here.